Description: 01-allow-systemd-resolved-selinux-config.patch
Index: refpolicy-2.20231119/policy/modules/system/systemd.te
===================================================================
--- refpolicy-2.20231119.orig/policy/modules/system/systemd.te
+++ refpolicy-2.20231119/policy/modules/system/systemd.te
@@ -1726,6 +1726,11 @@ manage_lnk_files_pattern(systemd_resolve
 manage_sock_files_pattern(systemd_resolved_t, systemd_resolved_runtime_t, systemd_resolved_runtime_t)
 init_runtime_filetrans(systemd_resolved_t, systemd_resolved_runtime_t, dir)

+# Allow systemd_resolved_t to read SELinux config and file contexts
+seutil_read_config(systemd_resolved_t)
+seutil_read_default_contexts(systemd_resolved_t)
+seutil_read_file_contexts(systemd_resolved_t)
+
 dev_read_sysfs(systemd_resolved_t)

 kernel_read_kernel_sysctls(systemd_resolved_t)
