Description: 02-allow-virt-urandom.patch
Index: refpolicy-2.20231119/policy/modules/services/virt.te
===================================================================
--- refpolicy-2.20231119.orig/policy/modules/services/virt.te
+++ refpolicy-2.20231119/policy/modules/services/virt.te
@@ -1127,6 +1127,8 @@ manage_files_pattern(virt_bridgehelper_t

 kernel_read_network_state(virt_bridgehelper_t)

+dev_read_urand(virt_bridgehelper_t)
+
 corenet_rw_tun_tap_dev(virt_bridgehelper_t)

 userdom_search_user_home_dirs(virt_bridgehelper_t)
